arm-software/embedded/docs/building-from-source.md

```diff
@@ -162,17 +163,21 @@ The same build directory can be used for both native and MinGW toolchains.
 
 ## Divergences from upstream
 
-See [patches](https://github.com/arm/arm-toolchain/-/blob/arm-software/embedded/patches)
-directory for the current set of differences from upstream.
+See [patches](patches) directory for the current set of differences from upstream.
 
 The patches for llvm-project are split between two folders, llvm-project and
 llvm-project-perf. The former are generally required for building and
 successfully running all tests. The patches in llvm-project-perf are optional,
 and designed to improve performance in certain circumstances.
 
-If not already applied, these must be done so manually before building, e.g.:
+Similarly, there are patches available for both picolibc and Newlib. The Picolibc
```

Comment:
Make sure capitalisation is correct and consistent. In this paragraph, picolibc and newlib have different capitalisations.